src/zero/module/aou.clj

```diff
@@ -1,23 +1,400 @@
 (ns zero.module.aou
-  "Process Arrays for the All Of Us project.")
+  "Process Arrays for the All Of Us project."
+  (:require [clojure.java.io :as io]
+            [clojure.data.json :as json]
+            [clojure.java.jdbc :as jdbc]
+            [clojure.set :as set]
+            [clojure.string :as str]
+            [zero.environments :as env]
+            [zero.module.all :as all]
+            [zero.references :as references]
+            [zero.service.cromwell :as cromwell]
+            [zero.service.postgres :as postgres]
+            [zero.service.gcs :as gcs]
+            [zero.util :as util]
+            [zero.wdl :as wdl]
+            [zero.zero :as zero])
+  (:import [java.time OffsetDateTime]
+           [java.util UUID]))
 
 (def pipeline "AllOfUsArrays")
 
-(defn update-workload!
-  "Use transaction TX to update WORKLOAD statuses."
-  [tx workload])
+(def workflow-wdl
+  "The top-level WDL file and its version."
+  {:release "Arrays_v1.9"
+   :top     "pipelines/arrays/single_sample/Arrays.wdl"})
+
+(def cromwell-label-map
+  "The WDL label applied to Cromwell metadata."
+  {(keyword (str zero/the-name "-aou"))
+   (wdl/workflow-name (:top workflow-wdl))})
+
+(def cromwell-label
+  "The WDL label applied to Cromwell metadata."
+  (let [[key value] (first cromwell-label-map)]
+    (str (name key) ":" value)))
+
+#_(def per-sample
```

Comment:
This comment looks old...can it be removed?

Reply:
removed!